pylint/pyreverse/utils.py

```diff
@@ -121,7 +115,7 @@ def __init__(self, mode):
                 print(f"Unknown filter mode {ex}", file=sys.stderr)
         self.__mode = __mode
 
-    def show_attr(self, node):
+    def show_attr(self, node: nodes.NodeNG | str) -> bool:
         """Return true if the node should be treated."""
         visibility = get_visibility(getattr(node, "name", node))
```

Comment:
```suggestion
        visibility = get_visibility(getattr(node, "name", node))
```

Unrelated, but this is quite a funny hack 😄